start-tests.sh

```diff
@@ -364,15 +364,15 @@ fi
 
 if [[ "$SKIP_UI_TEST" == false ]]; then
   # Displaying cypress environment variables, so we know all of the ones that are being passed successfully.
-  env | grep "cypress_" -i
+  # env | grep "cypress_" -i
```

Comment:
Please log messages to reming us to revert the 4 changes in this file.  See comment above for example.

Reply:
Updated this command to remove password env variables.